src/ElasticRepository.js

```diff
@@ -4,11 +4,11 @@ const { v4: uuid } = require("uuid");
 const debug = require("debug")("app:debug");
 class ElasticRepository {
   constructor() {
-    if (!process.env.AUDIT_INDEX) {
-      throw new Error("AUDIT_INDEX NOT SET");
-    }
-
-    this.client = new elasticsearch.Client({
+    if (!process.env.AUDIT_INDEX || !process.env.ELASTICSEARCH_HOST) {
+      // throw new Error("AUDIT_INDEX NOT SET");
+       console.log("AUDIT env not set")
+    }else{
+      this.client = new elasticsearch.Client({
```

Comment:
No console logs. Let it throw error.

Reply:
Then audit log will always have to live and collecting logs even for local dev setups it cant be turned off